File: Jul_17.js
//find the median of an array

function median(numbers) {
  var median = 0;
  numsLen = numbers.length;
  numbers.sort();

  if (numsLen % 2 === 0) {
    median = (numbers[numsLen / 2 - 1] + numbers[numsLen / 2]) / 2;
  } else {
    median = numbers[(numsLen - 1) / 2];
  }

  return median;
}

console.log(median([3, 5, 4, 4, 1, 1, 2, 3]));

File: Jul_17.py
#takes in two numbers and an operator, then returns the evaluated answer

import operator

ops = {
    "+": operator.add,
    "-": operator.sub,
    "*": operator.mul,
    "/": operator.truediv
}   

def maths(a,b, op_char):
  op_func = ops[op_char]
  return op_func(a, b)

print(maths(2, 5, "+"))


#------------------------------------
#find the median of a list of numbers

import statistics

numlist = [4, 8, 12, 62, 1, 6, 19, 14]

def median_num(nums):
  return statistics.median(nums)

print(median_num(numlist))